package-lock.json

```diff
@@ -51,27 +52,24 @@
         },
         "node_modules/@babel/code-frame": {
             "version": "7.12.11",
-            "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.11.tgz",
-            "integrity": "sha512-Zt1yodBx1UcyiePMSkWnU4hPqhwq7hGi2nFL1LeA3EUl+q2LQx16MISgJ0+z7dnmgvP9QtIleuETGOiOH1RcIw==",
             "dev": true,
+            "license": "MIT",
```

Comment:
This change is unwanted, Can I ask to retry using npm v 9.*.* ? It should not change unrelated packages adding license information AFIK.

Reply:
strange my npm version is 9.8.0 , it should produce the same lock file

what is yours ?

Reply:
I have 9.5.1, but I'm reading [here](https://github.com/npm/cli/issues/4263) that it is a common problem and sometimes depends on your env. Maybe it is worth to try [these steps](https://github.com/npm/cli/issues/4263#issuecomment-1673846177)?